v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -114,12 +120,11 @@ CockroachDB requires TCP communication on two ports:
     --advertise-addr=<node1 internal address> \
     --join=<node1 internal address>,<node2 internal address>,<node3 internal address> \
     --cache=.25 \
-    --max-sql-memory=.25 \
```

Comment:
This page has all the edits described by Aditya & Aayush in the 140k gdoc.  PTAL and let me know if this is an accurate edit based on that doc (might be easier to look at the generated preview pages under the 'Conversation' tab.